src/raglite/_database.py

```diff
@@ -287,6 +289,19 @@ def from_chunks(
         )
 
 
+@lru_cache(maxsize=1)
```

Comment:
Does caching make sense here? Every call to `create_database_engine` will create a new `Session`, right?